### PR TITLE
Older versions of Leap&SLE+non-UEFI also have other hotkey

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
## Ticket

- [poo#12410](https://progress.opensuse.org/issues/12410)

## Verification run

- [copland#1596#step/disable_grub_timeout/7](http://copland.arch.suse.de/tests/1596#step/disable_grub_timeout/7)

### CODE REVIEW

File: tests/installation/disable_grub_timeout.pm [hunk 1]
```diff
@@ -48,14 +48,11 @@ sub run {
     save_screenshot;    # needed because shortcuts change with the navigation
 
     # Select bootloader options tab
-    if ((is_sle && !sle_version_at_least('12-SP2')) || (is_leap && !leap_version_at_least('15'))) {
-        $cmd{bootloader} = 'alt-t';    # older versions
-    }
-    elsif (is_leap) {
+    if (is_leap && check_var('VERSION', '15')) {
```
Comment (okurz) [inline tests/installation/disable_grub_timeout.pm]: no, don't use a version specific check. We don't want to revisit this for '15.0' or '15-SP1'. What's wrong with the old code?

File: tests/installation/disable_grub_timeout.pm [hunk 2]
```diff
@@ -48,14 +48,11 @@ sub run {
...
         $cmd{bootloader} = 'alt-l';    # leap 15
     }
     else {
-        $cmd{bootloader} = check_var('UEFI', '1') ? 'alt-t' : 'alt-r';    # rest except uefi
+        $cmd{bootloader} = check_var('UEFI', '1') ? 'alt-t' : 'alt-r';    # UEFI or non-UEFI
```
Comment (okurz) [inline tests/installation/disable_grub_timeout.pm]: IMHO the comment is redundant anyway. There is a check for UEFI so no need to write "UEFI or non-UEFI".

File: tests/installation/disable_grub_timeout.pm (hunk 1 above)
Comment (SergioAtSUSE) [reply tests/installation/disable_grub_timeout.pm]: The problem of the old is that there is a too complicated if-else-statements when the only different version that doesn't match the pattern (UEFI->t, non-UEFI->r) is Leap 15. So I simplified the to a single if-statement.

Would it be better if I use:

```
if (is_leap && leap_version_at_least('15')
```

**Comment by SergioAtSUSE** [comment]
Superseeded by: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3762

**Pull request closed without merging**